code/modules/lighting/light_effect_cast.dm

```diff
@@ -66,14 +66,16 @@ var/light_post_processing = 1 // Use writeglobal to change this
 		alpha = initial(alpha)
 		animate(src, alpha = initial(alpha) - rand(30, 60), time = 2, loop = -1, easing = SINE_EASING)
 
-	for (var/mob/M in view(world.view, src))
-		M.check_dark_vision()
-
-	for(var/turf/T in view(2*light_range, src))
-		T.lumcount = -1
-		affecting_turfs += T
-		if (get_dist(T, location) <= distance_to_wall_illum && CHECK_OCCLUSION(T))
-			affected_shadow_walls += T
+	for (var/thing in view(world.view, src))
+		if (ismob(thing))
+			var/mob/M = thing
+			M.check_dark_vision()
+		if (isturf(thing))
+			var/turf/T = thing
+			T.lumcount = -1
+			affecting_turfs += T
```

Comment:
it's still doing it?

Reply:
See last commit, normally, it shoundn't. (`min(world.view, light_range)`)

Reply:
Nope I am retarded